.github/PULL_REQUEST_TEMPLATE/pull-request-template.md

```diff
@@ -10,11 +10,81 @@ assignees: ''
-- Result of automatic tests proving regression free.
+
+### Testing done
+
+<!-- Comment:
+Provide a clear description of how this change was tested.
+At minimum this should include proof that a computer has executed the changed lines.
+Ideally this should include an automated test or an explanation as to why this change has no tests.
+Note that automated test coverage is less than complete, so a successful PR build does not necessarily imply that a computer has executed the changed lines.
+If automated test coverage does not exist for the lines you are changing, **you must describe** the scenario(s) in which you manually tested the change.
+For frontend changes, include screenshots of the relevant page(s) before and after the change.
+For refactoring and code cleanup changes, exercise the code before and after the change and verify the behavior remains the same.
+-->
+
+### Proposed upgrade guidelines
+
+N/A
+
+### Localizations
+
+<!-- Comment:
++ Be sure any localization files are moved to /msg/ files.
++ Please describe here which language has been translated by you.
++ English text's are mandatory for new entries.
+-->
+
+- [ ] English
+- [ ] German
+
+### Submitter checklist
+
+- [ ] The Github issue, if it exists, is well-described.
+- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood.
+  - The changelog generator for plugins uses the **pull request title as the changelog entry**.
+  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during the upgrade.
+- [ ] There is automated testing or an explanation that explains why this change has no tests.
+- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
+- [ ] Any localizations are transferred to /msg/ files.
+<!-- TBD
+- [ ] Changes in the interface are documented also as [examples](docs/examples/readme.md).
+-->
+
+### Maintainer checklist
+
+Before the changes are marked as `ready-for-merge`:
+
+<!-- TBD, maybe done in GitHub rules
+- [ ] There is at least one (1) approval for the pull request and no outstanding requests for change.
+- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
+-->
+- [ ] Changelog entries in the **pull request title** and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
+<!-- TBD, maybe done in GitHub rules
+- [ ] Proper changelog labels are set so that the changelog can be generated automatically. See also [release-drafter-labels](...).
+-->
+- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](...)).
```

Comment:
We don't have this label _upgrade-guide-needed_ yet, so please mention _documentation_ instead or explain, why we would need another label.

Reply:
Have we some labels? A good example is here: 
https://github.com/jenkinsci/.github/blob/ce466227c534c42820a597cb8e9cac2f2334920a/.github/release-drafter.yml#L9-L50
I will remove (comment out) the line from Template

Reply:
See the current list here:
https://github.com/siemens/CtrlppCheck/issues/labels